app/helpers/my_facilities_helper.rb

```diff
@@ -4,7 +4,7 @@ def preserve_query_params(params, preserve_list)
   end
 
   def percentage(numerator, denominator)
-    return "—" if denominator.nil? || denominator.zero? || numerator.nil?
```

Comment:
Was this not handling the 0 -> "-" conversion properly?

Reply:
I changed this because percentage values within pills showed `0%` and others showed `-`. It was confusing to see that next to a `-` in the number column. Sample screenshot below:

![image](https://user-images.githubusercontent.com/16785131/98393031-bc019b00-2026-11eb-87be-4ca3a2389bb3.png)